.yarn/versions/53dfe063.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  "@nytimes/react-prosemirror": patch
```

Comment:
Should we make this a minor version change, like #83? It's also "not a breaking change", but folks may have strict type checking or linting rules that error with unnecessary null checks, which would result in broken pipelines after incorporating these changes

Reply:
that's a great suggestion. will update!